### DISCUSSION TITLE
fail pytest

### DISCUSSION THREAD

**Discussion by ArtemIsmagilov** [pallets/quart#287] [Q&A]
Task <Task pending name='Task-13' coro=<ASGIHTTPConnection.handle_request() running at /home/artem/PycharmProjects/mm-yc-notify/venv/lib/python3.12/site-packages/quart/asgi.py:98> cb=[_wait.<locals>._on_completion() at /usr/lib/python3.12/asyncio/tasks.py:524]> got Future <Future pending cb=[Protocol._on_waiter_completed()]> attached to a different loop
Environment:

- Python version: 3.12
- Quart version: 0.19.3

**Comment by ThiefMaster** [comment]
Not sure how you expect any help if you do not even post code to reproduce this...
